src/canmatrix/canmatrix.py

```diff
@@ -587,10 +587,14 @@ class ArbitrationId(object):
     compound_extended_mask = (1 << 31)
 
     id = attr.ib(default=None)
-    extended = attr.ib(default=None)
+    extended = attr.ib(default=False)  # type: bool
 
     def __attrs_post_init__(self):
-        if self.extended is None or self.extended:
+        if self.extended is None:
+            # Mimicking old behaviour for now -- remove in the future
+            self.extended = True
+            logger.warning("Please set 'extended' attribute as a boolean")
```

Comment:
This should be a warning rather than being logged.  I'm not sure but maybe there's something in https://pypi.org/project/regret/ that would take it to a higher level yet.

Reply:
Done